src/index.js

```diff
@@ -144,10 +142,10 @@ const App = () => {
                     <ToDoItems todos={sortedTodos} onCompleteItem={handleCompleteItem} />
                 </tbody>  
             </table>
-            <input type='text' placeholder='event name' ref={toDoInput} />
-            <input type="date" min={tomrISO} ref={dueDateInput} /> <button onClick={handleAddItem}> Add </button>
+            <input type='text' placeholder='event name' value={todoInputValue} onChange={e => setTodoInputValue(e.target.value)}  />
+            <input type="date" min={tomrISO} value={dateInputValue} onChange={e => setDateInputValue(e.target.value)} /> <button onClick={handleAddItem} disabled={isAddButtonDisabled}> Add </button>
```

Comment:
A form element becomes “controlled” if you set its value via a prop.
Use `value={dateInputValue}` to make it a controlled element, and able to clear it using its set function.